.github/actions/run-python-test-set/action.yml

```diff
@@ -132,7 +132,7 @@ runs:
         fi
         if [[ "${{ inputs.run_in_parallel }}" == "true" ]]; then
           # -n16 uses sixteen processes to run tests via pytest-xdist
```

Comment:
Let's adjust this comment as well (we probably don't need to have an exact number in it)

Reply:
done